packages/availity-angular-kit/package.json

```diff
@@ -64,10 +64,7 @@
     "babel-eslint": "^7.2.3",
     "cross-env": "^5.0.4",
     "eslint": "^4.4.1",
-    "eslint-config-availity": "^2.2.0",
-    "eslint-plugin-import": "^2.7.0",
```

Comment:
These are peerDeps of `eslint-config-availity`. It will complain that they are not installed.

Reply:
i'll fix